creator-node/src/services/stateMachineManager/stateReconciliation/types.ts

```diff
@@ -1,15 +1,37 @@
 // Issue sync request job
+
+export type SyncRequestAxiosData = {
```

Comment:
was just about to ask if you could update this type but you already got it -- nice!